### PR TITLE
Feat typography

### PR DESCRIPTION
**Pull request by hinanshisuthar**
Text Utilities
- headings
- small text
- gray text
- alingned text
- weighted text
- strikethrough text

[Documentation - 8 February 2022 - Watch Video](https://www.loom.com/share/c8b5e992ce7f4d20a97d33449022dc86)

### CODE REVIEW

**Review by shuklahimanshu81 — COMMENTED** [review]
Clean code structure hinanshi. Good to merge.

File: docs/docs.css [hunk 1]
```diff
@@ -183,6 +183,13 @@ article .eg-container {
...
+#typography .text-left,
+#typography .text-center,
+#typography .text-right {
+    width: 100%;
+}
+
+
 /* media queries */
 
 @media screen and (max-width: 275px) {
```
Comment (Hamdrive) [inline docs/docs.css:195]: Why have you gone for very small and unique screen sizes?
Would be better if you used the commonly used breakpoints

File: docs/docs.html [hunk 2]
```diff
@@ -1004,6 +1008,145 @@ <h3>Example of Input with validation and error</h3>
...
+
+                    <div class="m-md"></div>
+                    <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28207%2C207%2C207%2C1%29&t=one-light&wt=none&l=auto&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=17px&ph=20px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253C%21--%2520copy%2520and%2520paste%2520the%2520following%2520code%2520in%2520your%2520html%2520file.%2520Give%2520appropriate%2520class%2520namings%2520as%2520shown%2520in%2520the%2520snippet%2520below%2520--%253E%250A%250A%253Cp%2520class%253D%2522text-lowercase%2522%253ELOWERCase%2520teXt%253C%252Fp%253E%250A%253Cp%2520class%253D%2522text-uppercase%2522%253Euppercase%2520text%253C%252Fp%253E%250A%253Cdiv%2520class%253D%2522text-capitalize%2522%253Ecapitalise%2520text%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522text-grey%2522%253EGrey%2520Text%253C%252Fdiv%253E%250A%253Csmall%253Esmall%2520text%253C%252Fsmall%253E%250A%253Cdiv%2520class%253D%2522text-strikethrough%2522%253Estrikethrough%2520text%253C%252Fdiv%253E"
+  style="width: 100%; height: 200px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
```
Comment (Hamdrive) [inline docs/docs.html:1145]: Use prettier or any other code formatter to fix this

**Review by Hamdrive — COMMENTED** [review]
Great work done.
Have suggested some fixes for media query and formatting

File: docs/docs.css (hunk 1 above)
Comment (hinanshisuthar) [reply docs/docs.css:195]: Yes I used the common breakpoints only first but as vikram suggested wherever your design is breaking you could add a media query so did that.

File: docs/docs.html (hunk 2 above)
Comment (hinanshisuthar) [reply docs/docs.html:1145]: Yep cool done.

**Pull request merged**